docs/pages-for-subheaders/aws-marketplace-payg-integration.md

```diff
@@ -0,0 +1,276 @@
+---
+title: AWS Marketplace Pay-as-you-go (PAYG) Integration
+---
+
+<head>
+  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/pages-for-subheaders/aws-marketplace-payg-integration"/>
+</head>
+
+## Overview
+
+Rancher Prime integrates with the AWS Marketplace as a pay-as-you-go (PAYG) offering. This brings the value of running and managing Kubernetes environments to AWS customers, with the benefit of a new pay monthly pricing model available through the AWS Marketplace. This listing will enable you to manage any CNCF-certified Kubernetes
+distribution in AWS, on-prem, or at the edge. To learn more, see our [AWS Marketplace offering](https://aws.amazon.com/marketplace/pp/prodview-f2bvszurj2p2c?sr=0-2&ref_=beagle&applicationId=AWSMPContessa).
+
+## Limitations
+
+## How to Use
+
+1. Complete the [prerequisite steps](../integrations-in-rancher/cloud-marketplace/aws-marketplace-payg-integration/rancher-prime-aws.md).
+2. [Install the Rancher Prime PAYG offering on the AWS Marketplace](../integrations-in-rancher/cloud-marketplace/aws-marketplace-payg-integration/installing-rancher-prime.md).
+
+## FAQ
+
+The following is a list of frequently asked questions.
+
+### Marketplace Listing
+
+**What is the Rancher Prime listing on the AWS Marketplace?**
+
+By selecting the Rancher Prime listing from the AWS Marketplace, customers can deploy Rancher to their Kubernetes environment with the advantage of having monthly billing via AWS.
+
+**Where do I find the Rancher Prime listings?**
+
+The listings can be found in the AWS Marketplace, there are two listings:
+- Rancher Prime (No EU, EEA, or UK Orders)
+- Rancher Prime (EU, EEA, or UK Orders)
+
+**Why are there 2 listings, which one should I use?**
```

Comment:
the titles of the listings have changed since they were initially published and are now:
[Rancher Prime](https://aws.amazon.com/marketplace/pp/prodview-f2bvszurj2p2c)   and  [Rancher Prime (EMEA Orders Only)](https://aws.amazon.com/marketplace/pp/prodview-ocgjwd5c2aj5i)
these naming changes should apply to the question above and the "2 listings" question just below